dir.traversal.targets

```diff
@@ -32,11 +32,14 @@
         <AdditionalProperties Condition="'%(Project.InputOSGroup)' != ''">InputOSGroup=%(Project.InputOSGroup);%(Project.AdditionalProperties)</AdditionalProperties>
       </Project>
       <Project>
+        <AdditionalProperties Condition="'%(Project.BuildAllOSGroups)' != ''">BuildAllOSGroups=%(Project.BuildAllOSGroups);%(Project.AdditionalProperties)</AdditionalProperties>
```

Comment:
FYI @mellinoe is also making this same change so I suspect which ever one of you guys gets in first the other will need to merge this out.